File: objectModel/Java/objectmodel/src/main/java/com/microsoft/commondatamodel/objectmodel/persistence/cdmfolder/types/projections/OperationCombineAttributes.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See License.txt in the project root for license information.

package com.microsoft.commondatamodel.objectmodel.persistence.cdmfolder.types.projections;

import com.fasterxml.jackson.databind.JsonNode;

import java.util.List;

/**
 * OperationCombineAttributes class
 */
public class OperationCombineAttributes extends OperationBase {
    private List<String> select;
    private JsonNode mergeInto;

    public List<String> getSelect() {
        return select;
    }

    public void setSelect(final List<String> select) {
        this.select = select;
    }

    public JsonNode getMergeInto() {
        return mergeInto;
    }

    public void setMergeInto(final JsonNode mergeInto) {
        this.mergeInto = mergeInto;
    }
}

File: objectModel/TypeScript/Persistence/CdmFolder/types/Projections/OperationCombineAttributes.ts
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See License.txt in the project root for license information.

import { TypeAttribute } from '../TypeAttribute';
import { OperationBase } from './OperationBase';

/**
 * OperationCombineAttributes class
 */
export abstract class OperationCombineAttributes extends OperationBase {
    public select: string[];
    public mergeInto: TypeAttribute;
}
